application.js

```diff
@@ -24,15 +22,14 @@ app.use(express.static(__dirname + '/public'));
 // Note: important that this is added just before your own Routes
 app.use(mbaasExpress.fhmiddleware());
 
-// fhlint-begin: custom-routes
+app.use('/hello', require('./lib/hello.js'));
```

Comment:
I'd guess as an example it's still valid. Not really looking to address existing lack of tests here (since that's an issue that affects pretty much all of our templates), just want some sort of basic CI for sanity.